common/changes/office-ui-fabric-react/satishkumaragarwal-customRenderCoinIssue_2018-06-20-16-20.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixing issue with render coin on persona component",
+      "type": "patch"
```

Comment:
only commit this change file. remove the others with 'type': "none"

Reply:
Done. Thanks.

Reply:
Sorry, I should have said this before. But please update the "comment" above to describe the actual bug. This text actually gets used in our release notes, so knowing generally what the bug was is really helpful.